cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/ExportXmlCompare.java

```diff
@@ -578,6 +578,44 @@ static ComparisonResult ignoringOperationalLimitIds(Comparison comparison, Compa
         return result;
     }
 
+    static ComparisonResult ignoringLoadAreaIds(Comparison comparison, ComparisonResult result) {
```

Comment:
@miovd I am absolutely not sure that this is how it should be done. Maybe for ID but I feel like there should be another way for resource?